pages/protect-your-data.jsx

```diff
@@ -0,0 +1,132 @@
+var React = require('react');
+var Illustration = require('../components/illustration.jsx');
+var ActivitySection = require('../components/activity-section.jsx');
+var CCLicenseNote = require('../components/cc-license-note.jsx');
+var Router = require('react-router');
+var Link = Router.Link;
+
+var Intro = React.createClass({
+  render: function () {
+    return (
+      <div>
+        <h1>Protect Your Data</h1>
+        <section className="intro">
+          <Illustration
+          height={204} width={204}
+          src1x="/img/pages/protect-your-data/photo-protect-your-data.png"
+          src2x="/img/pages/protect-your-data/photo-protect-your-data@2x.png"
+          alt=""
+          className="img-circle">
+            <h2>Engage learners around the topic of privacy in this six-part module, developed in conjunction with Hive Toronto and the Mozilla Privacy Team.</h2>
+          </Illustration>
+        </section>
+      </div>
+    );
+  }
+});
+
+
+var LearningObjectives = React.createClass({
+  render: function () {
+    return (
+      <section className="row web-lit-basics">
+        <div className="col-sm-12">
+          <h2>Learning Objectives</h2>
+          <p>
+            These activities will help your learners understand how to take control of their privacy in a networked world. Complete the activities in sequence, or mix and match. <a href="http://discourse.webmaker.org/category/clubs">Visit our discussion forum</a> to get help and share your experience.
```

Comment:
The w3c [discourages verbs in link text](http://www.w3.org/QA/Tips/noClickHere), but as you can see from this comment, I do not particularly care. That said, if we wanted to be more internet savvy, we could leave "Visit our" out of the hyperlink to our discussion forum.